src/mots/utils.py

```diff
@@ -39,3 +38,32 @@ def parse_real_name(real_name):
         return match.groupdict()
     else:
         return {"name": None, "info": None}
+
+
+class Disk:
+    """Utilities to read and write settings files and directories to disk."""
+
+    def __init__(self):
+        self.home = Path.home()
```

Comment:
Should the default be in the homedir or in the repo root? Feels like we don't need a user-level `mots` config, only one specific to the tree.

Reply:
So this is for mots-specific settings, in contrast with a "mots config" (which is a `mots.yaml` file in the repo root). They are two separate things. Things you would put in this config file generally have no effect on the repo (e.g. "debug mode" or "log file path".)